### PR TITLE
Custom pathing for evaluations

### PR DESCRIPTION
**Pull request by davidslater**
Also relaxes attack API requirements.

### CODE REVIEW

**Comment by ng390** [comment]
Testing looked good for --check runs with both local repositories and with "external_github_repo_pythonpath" set.

**Comment by davidslater** [comment]
@ng390 ready for another review - I made the updates with git config.

File: armory/utils/config_loading.py [hunk 1]
```diff
@@ -101,8 +101,10 @@ def load_attack(attack_config, classifier):
     attack_module = import_module(attack_config["module"])
     attack_fn = getattr(attack_module, attack_config["name"])
     attack = attack_fn(classifier, **attack_config["kwargs"])
-    if not isinstance(attack, Attack):
```
Comment (ng390) [inline armory/utils/config_loading.py:104]: This if statement is needed

**Comment by davidslater** [comment]
Ready for re-review - added the if back in to the warning.

**Pull request merged**